packages/core/src/docdb/explorer/dbClusterNode.ts

```diff
@@ -170,6 +170,7 @@ export class DBClusterNode extends DBResourceNode {
     }
 
     override refreshTree(): void {
+        getLogger().info(`(DBClusterNode) Refreshing tree for instance: ${this.arn}`)
```

Comment:
do we need these as info? What gain does the user see in their logs from seeing the tree is refreshed?

Reply:
I think we can just add it as the debug would be good, that way we can have it if something needs to debugged with tree refreshing but regular users won't see it